packages/bsky/src/api/app/bsky/util/feed.ts

```diff
@@ -100,3 +100,10 @@ export class FeedKeyset extends TimeCidKeyset<FeedRow> {
     return { primary: result.sortAt, secondary: result.cid }
   }
 }
+
+// For users with sparse feeds, avoid scanning back further than two weeks
+export const getFeedDateThreshold = (days = 14) => {
```

Comment:
did we want to consider 7 days instead?

or maybe within 7 days of the supplied cursor value?

Reply:
Ooh the latter option seems solid: a single page can only get up to a week of content.  That way you could go pretty far back in time as long as your feed doesn't eventually blackhole.  Going to implement that 👍

Reply:
@dholms made those changes, wouldn't mind another peek.